src/infrastructure/fragmenter.rs

```diff
@@ -50,7 +51,7 @@ impl Fragmentation {
     ///
     /// So for 4000 bytes we need 4 fragments
     /// [fragment: 1024] [fragment: 1024] [fragment: 1024] [fragment: 928]
-    pub fn total_fragments_needed(payload_length: u16, fragment_size: u16) -> u16 {
+    pub fn fragments_needed(payload_length: u16, fragment_size: u16) -> u16 {
         let remainder = if payload_length % fragment_size > 0 {
             1
```

Comment:
Should this be a constant for clarity?

Reply:
Nah, because it means just one fragment. Don't see the reason to move it into a constant.

Reply:
I am closing this because I think it is unnecessarily.